newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -77,12 +77,14 @@ fun Routing.createSongRoutes() {
                     )
                 }
                 get("stream") {
+                    val streamData = songRepository.generateAudioStreamData(
+                        songId = songId,
+                    )
+                    streamData.cookies.forEach { cookie ->
+                        response.cookies.append(name = cookie.key, value = cookie.value, domain = "newm.io")
+                    }
```

Comment:
Since now we set the cookies in the response, do we still need to send the cookies on the Response payload? Or can we now only send back the the URL on the payload?

Reply:
If the cookies in the response work, I'll remove the cookies and only send back the url.  Sorry for the iteration - its been hard to test this locally.